docker/test/develop.yml

```diff
@@ -0,0 +1,42 @@
+# This needs at least compose 1.6.0
+version: '2'
+
+services:
+  postgresdata:
+    image: postgres:9.5
+    command: /bin/true
+
+  postgres:
+    image: postgres:9.5
+    volumes_from:
+      - postgresdata
+    environment:
+      POSTGRES_PASSWORD: myhuginnpassword
+      POSTGRES_USER: huginn
+
+  huginn_test_postgres:
+    build:
+      context: ../../
+      dockerfile: docker/test/Dockerfile
+    restart: always
+    extends:
+      file: environment.yml
+      service: huginn_base
+    environment:
+      DATABASE_ADAPTER: postgresql
+      DATABASE_SOCKET:
+      POSTGRES_PORT_5432_TCP_ADDR: postgres
+      POSTGRES_PORT_5432_TCP_PORT: 5432
+      RAILS_ENV: test
+      DO_NOT_SEED: 1
+    ports:
+      - 3000:3000
+    links:
+      - postgres
+    volumes:
+      - ../../app:/app/app
```

Comment:
I think the main reason was to exclude the `vendor` directory to avoid local and docker container bundler conflicts- I'll try to find a more elegant approach (maybe using `.dockerignore`)